### ISSUE TITLE
Model Training

### ISSUE THREAD

**Issue by joonesmks** [pjh5672/YOLOv1#2]
Currently, the README.md file only outlines how to use the pre-trained weights to train a custom model. 

It would be great, if the README would have a subsection that briefly outlines how to train the model on the PASCAL-VOC dataset in order to obtain training data like loss, etc.

**Comment by pjh5672** [comment]
@joonesmks  Thank for your interest 👍  I will add training description in more details in README.md soon. 
In other hand, you can train YOLOv1 model with [2-1. Model Training] subsection. For example of VOC training, you can set VOC data directory to "PATH" in "data/voc.yaml" in this repo. (PASCAL-VOC data directory must contain the sub-directory of "images/train2007", "images/train2012", "images/val2007", and "images/val2012".)

**Comment by pjh5672** [comment]
@joonesmks for download proper VOC dataset, you can use this scripts with L79-84 from "https://github.com/ultralytics/yolov5/blob/master/data/VOC.yaml"
